src/julia.h

```diff
@@ -1829,8 +1829,8 @@ typedef struct _jl_task_t {
     jl_value_t *result;
     jl_value_t *logstate;
     jl_function_t *start;
+    uint16_t sticky; // 0 means this Task can be migrated to a new thread
```

Comment:
uint16 here is somewhat arbitrary. It works with uint8 too (see the saturation check) but I thought 255 tasks are not super crazily many tasks.